.travis.yml

```diff
@@ -11,4 +11,18 @@ install:
   - pip install -r requirements_dev.txt
 # command to run tests
 script:
-  - make test
+- make test
+
+before_deploy: sed -i "s/__version__ = \".*\"/__version__ = \"$TRAVIS_TAG\"/g" azkaban_cli/__version__.py
+
+deploy:
+  provider: pypi
+  user: $PYPI_USER
+  password: $PYPI_PASSWORD
```

Comment:
Thank you for this great work!

We should use encrypted password as environment variable
```
deploy:
  provider: pypi
  user: "Your username"
  password:
    secure: "Your encrypted password"
```

Reply:
Done, you will have to encrypt password using [travis encrypt](https://docs.travis-ci.com/user/encryption-keys#usage)